src/ngx_lua_resty_lmdb_module.c

```diff
@@ -539,7 +539,7 @@ static ngx_int_t ngx_lua_resty_lmdb_init(ngx_cycle_t *cycle)
     if (ngx_lua_resty_lmdb_validate(cycle, lcf) != NGX_OK) {
         ngx_lua_resty_lmdb_close_file(cycle, lcf);
 
-        ngx_log_error(NGX_LOG_WARN, cycle->log, 0,
+        ngx_log_error(NGX_LOG_NOTICE, cycle->log, 0,
```

Comment:
I want this to be more visible as wiping the database is a destructive operation. Why would this message occur in our tests anyway? We are not testing the upgrade in integration tests.

Also, could you test by changing the `.requirments` file to point to this branch and shows the test passes? I want to make sure this PR is good before merging.

Reply:
When kong first start, there is no tag in lmdb, and we have set validation tag in nginx-kong.conf, so the C module will wipe the lmdb database and leave a log message.

If the log is warning level, it will break the test case in `spec/02-integration/02-cmd/02-start_stop_spec.lua`.

```
assert.logfile().has.no.line("[warn]", true)
```

Reply:
PR https://github.com/Kong/kong/pull/12026 is using this branch.

Reply:
Perhaps we could add another condition:
if the lmdb database is empty then write the tag into database and don't think it is an warning.

Reply:
done.

Reply:
> if the lmdb database is empty then write the tag into database and don't think it is an warning.

THis should not be necessary, for our use case we can assume lack of tag = empty database. We do not concern compatibility with other LMDB using applications.